discord/ext/commands/core.py

```diff
@@ -715,8 +715,7 @@ async def _parse_arguments(self, ctx):
                     except RuntimeError:
                         break
 
-        if not self.ignore_extra:
-            if not view.eof:
+        if not self.ignore_extra and not view.eof:
                 raise TooManyArguments('Too many arguments passed to ' + self.qualified_name)
```

Comment:
Indentation error for some reason.
```suggestion
            raise TooManyArguments('Too many arguments passed to ' + self.qualified_name)
```